server/utils/azureAppInsights.ts

```diff
@@ -40,7 +40,7 @@ export function buildAppInsightsClient(name = defaultName()): TelemetryClient {
     defaultClient.addTelemetryProcessor(({ tags, data }, contextObjects) => {
       const operationNameOverride = contextObjects.correlationContext?.customProperties?.getProperty('operationName')
       if (operationNameOverride) {
-        tags['ai.operation.name'] = data.baseData.name = operationNameOverride // eslint-disable-line no-param-reassign,no-multi-assign
+        tags['ai.operation.name'] = data.baseData.name = operationNameOverride // eslint-disable-line no-multi-assign
```

Comment:
Suggest changing to // eslint-disable-next-line no-multi-assign on the line above to aid readability.

Reply:
Good call!